fastlane/Fastfile

```diff
@@ -125,6 +126,9 @@ GLOTPRESS_TO_ASC_METADATA_LOCALE_CODES = {
   'zh-tw' => 'zh-Hant'
 }.freeze
 
+require_relative 'lib/helpers'
+import 'lib/helpers.rb'
```

Comment:
I think using both `require_relative` and `import` is redundant?

Though I always forget about the difference between the two (I know `require_relative` is from Ruby core, while iinm `import` is from _fastlane_, and I think each has it's own pros/cons, like `import` knowing about `lanes` and being able to call `UI` while already knowing we're within the `Fastlane::` module, but otoh not providing whatever feature that `require_relative` provides I can't remember which…)

I also know we have a mix of `require_relative` and `import` in e.g. the Tumblr Fastfile and that I've meant to better understand the difference between the two and clean up the mess to make it more consistent at some point (but never got to it)…

Reply:
It's been some days, so I don't remember exactly why I used this syntax. I think I copied it from Tumblr after getting a failure when using only `require_relative`.

I tested it locally again today. 

#### `require_relative` only

Fails because it cannot access the `USER_ENV_FILE_PATH` constant, which is defined in `Fastfile`.

```
[11:29:46]: Applied configuration
+---------------------------------------------------------+
|                      Lane Context                       |
+------------------+--------------------------------------+
| DEFAULT_PLATFORM | ios                                  |
| PLATFORM_NAME    | ios                                  |
| LANE_NAME        | ios configure_code_signing_app_store |
+------------------+--------------------------------------+
[11:29:46]: fastlane finished with errors

Looking for related GitHub issues on fastlane/fastlane...

Found no similar issues. To create a new issue, please visit:
https://github.com/fastlane/fastlane/issues/new
Run `fastlane env` to append the fastlane environment to your issue
bundler: failed to load command: fastlane (/Users/gio/Developer/a8c/pcios/vendor/bundle/ruby/3.2.0/bin/fastlane)
/Users/gio/Developer/a8c/pcios/fastlane/lib/helpers.rb:11:in `get_required_env!': [!] uninitialized constant USER_ENV_FILE_PATH (NameError)

def get_required_env!(key, env_file_path: USER_ENV_FILE_PATH)
                                          ^^^^^^^^^^^^^^^^^^
	from /Users/gio/Developer/a8c/pcios/fastlane/lib/helpers.rb:7:in `block in require_env_vars!'
```

#### `import` only

Works.

#### `required_relative` and `import`

Works. _We knew that already_.

So, looks like there's more to learn, at least for me. To be fair, I don't like how my implementation accesses constants that are defined in a different file without importing them. But I cut some corners in the interest of getting this done and having methods that are not too cumbersome to call.

I suppose there's a design we could achieve that moves all the constant at the helper level, with an additional method to call `DotEnv.load`. But... I like having the files in `fastlane/lib` be generic enough to be copy pasted to other process. But... that's a bit of a smell too. If we find ourselves copy pasting, maybe we should consider pushing these helper in `release-toolkit` or a new dedicated helper gem.